package.json

```diff
@@ -48,8 +48,18 @@
       "description": "How long to show successful build status, in seconds. Failed status will be shown indefinitely. Negative means forever",
       "type": "number",
       "default": "10"
+    },
+    "showSemanticView": {
+      "title": "Show semantic view",
+      "description": "Show semantic view (outline) for typescript content",
+      "type": "boolean",
+      "default": false
     }
   },
+  "deserializers": {
+    "atomts-semantic-view/SemanticView": "deserializeSemanticView",
+    "atomts-semantic-view/SemanticViewPane": "deserializeSemanticViewPane"
```

Comment:
:warning: There doesn't seem to be `deserializeSemanticViewPane` anywhere.

Reply:
ahh, this is a leftover from something I reverted -- will be removed